maintenance/manageInactiveWikis.php

```diff
@@ -51,7 +51,7 @@ public function execute(): void {
 		foreach ( $res as $row ) {
 			$dbName = $row->wiki_dbname;
 			$remoteWiki = $remoteWikiFactory->newInstance( $dbName );
-			$inactiveDays = (int)$this->getConfig()->get( ConfigNames::StateDays )['inactive'];
+			$inactiveDays = (int)$this->getConfig()->get( ConfigNames::StateDays )['default']['inactive'];
```

Comment:
I wonder if this should be no-edits?

Reply:
I think so? I'll do some thinking, but also need to look to make sure we're not breaking any previously implemented assumptions about the state from when there was only one track.

Reply:
If the `no_edits_inactivity < default_inactivity`, but we only do checking for when it's default, then the no edits stuff won't trigger, right?

Reply:
Updated to no-edits, but we should take a look to see if that doesn't break and assumptions and cause it to early close any wikis it shouldn't. A bit late for me to be doing a thoughtful look right now though.

Reply:
cc @Universal-Omega